okapi-core/src/main/java/org/folio/okapi/managers/InternalModule.java

```diff
@@ -330,6 +337,24 @@ public static ModuleDescriptor moduleDescriptor(String okapiVersion) {
         + "    \"permissionsRequired\" : [  \"okapi.proxy.tenants.interfaces.get\" ], "
         + "    \"permissionsRequiredTenant\" : [ ], "
         + "    \"type\" : \"internal\" "
+        + "   }, {"
+        + "    \"methods\" :  [ \"GET\" ],"
+        + "    \"pathPattern\" : \"/_/proxy/tenants/{tenantId}/timers\","
+        + "    \"permissionsRequired\" : [  \"okapi.proxy.tenants.timers.list\" ], "
+        + "    \"permissionsRequiredTenant\" : [ ], "
+        + "    \"type\" : \"internal\" "
+        + "   }, {"
+        + "    \"methods\" :  [ \"PATCH\" ],"
+        + "    \"pathPattern\" : \"/_/proxy/tenants/{tenantId}/timers\","
+        + "    \"permissionsRequired\" : [  \"okapi.proxy.tenants.timers.patch\" ], "
+        + "    \"permissionsRequiredTenant\" : [ \"okapi.proxy.self.timers.patch\" ], "
```

Comment:
`permissionsRequiredTenant` is used only here, no elsewhere. What does it do exactly?

Reply:
I thought it wasn't necessaryu to use it elsewhere as this is same tenant and other operations are read-only.

If you ask what permissionsRequiredTenant is, then it is the effective permission used if the operating tenant is the same as the tenant in the path.. Eg if user can only modify timer in his/her library (self).. The other permission is used for a tenant that operates on ALL tenants.. Eg like a sysadmin..